dev-requirements.txt

```diff
@@ -4,4 +4,5 @@ pytest-django>=4.5.2
 pytest-cov
 django-debug-toolbar
 sphinx
-pre-commit
\ No newline at end of file
+pre-commit
+tqdm
```

Comment:
while you're at it: could you check if we're still requiring `gensim` and comment it out? we should make it an optional install since we're not planning to run the other version of this script